tests/rpc_equivalence.rs

```diff
@@ -984,26 +984,57 @@ async fn test_get_historical_block(helios: &RootProvider, expected: &RootProvide
     Ok(())
 }
 
+async fn test_get_too_old_block(helios: &RootProvider, expected: &RootProvider) -> Result<()> {
+    // Fetch the latest block number from the trusted provider
+    let latest_block = expected.get_block_number().await?;
+    let latest_block_num = latest_block;
+
+    // Calculate a block number that is guaranteed to be outside the EIP-2935 ring buffer
+    let old_block_num = latest_block_num - 20000;
```

Comment:
let's use `saturation_sub` here to prevent underflows if we run these tests on a new network (like a testnet)